pkg/api/types.go

```diff
@@ -3023,6 +3023,10 @@ type NodeList struct {
 type NamespaceSpec struct {
 	// Finalizers is an opaque list of values that must be empty to permanently remove object from storage
 	Finalizers []FinalizerName
+
+	// NetworkPolicy describes the use of NetworkPolicy objects in this namespace. If not specified, then default policies will be applied and NetworkPolicy objects in the Namespace will be ignored
+	// +optional
+	NetworkPolicy *NamespaceNetworkPolicy
```

Comment:
Whoa... I was not expecting this.  Is network policy a core API?  Seems like this violates Brian's layering doc.